Backend_Mapanimal/Entities/Race.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Backend_Mapanimal.Entities
+{
+    public class Race
+    {
+        public int Id { get; set; }
```

Comment:
Please do private all the setters for the object attributes, we will respect the encapsulation principle of OOP